packages/data-stores/src/utils/index.ts

```diff
@@ -1 +1,14 @@
-export * from './wpcom-wrapper';
+/**
+ * External dependencies
+ */
+import wpcomProxyRequest from 'wpcom-proxy-request';
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'data-stores:utils:wpcom-wrapper' );
+
+wpcomProxyRequest( { metaAPI: { accessAllUsersBlogs: true } }, ( error: Error ) => {
+	if ( error ) {
+		throw error;
+	}
+	debug( 'Proxy now running in "access all user\'s blogs" mode' );
+} );
```

Comment:
The previous `wpcom-wrapper` was side-effectful, meaning this is inaccurate:

https://github.com/Automattic/wp-calypso/blob/1404ede98f12cccd7dd040c7f2a126b99c080c62/packages/data-stores/package.json#L10

That becomes more apparent and dangerous with this change.

Let's keep this side-effect free and contain this in a function:

```ts
export function requestAllBlogsAccess() {
  return wpcomProxyRequest( { metaAPI: { accessAllUsersBlogs: true } } );
}
```

This functionality is specific to wpcomProxyRequest and quite common, it might make sense to include it in the library.